label_sync docs (#6535)
kyma-project/test-infra: docs/prow/label_sync.md

# Label_sync
Label_sync updates or migrates github labels on repositories in a Github organisation based on a YAML file. It is trigged as a `ci-prow-label-sync` Prow job.

> Label Configuration can be found [here](https://github.com/kyma-project/test-infra/blob/main/prow/labels.yaml).

Reviewer: ```suggestion
For Label Configuration see the [labels](https://github.com/kyma-project/test-infra/blob/main/prow/labels.yaml) file.
```

> More excessive documentation and a source code is available in the Kubernetes `test-infra` [repository](https://github.com/kubernetes/test-infra/tree/master/label_sync).

Reviewer: ```suggestion
See also the more extensive documentation and the source code in the [Kubernetes `test-infra` repository](https://github.com/kubernetes/test-infra/tree/master/label_sync).
```
